phpmyfaq/src/phpMyFAQ/Controller/Frontend/AbstractFrontController.php

```
@@ -71,14 +82,14 @@ protected function getHeader(Request $request): array
                 : Translation::get(key: 'msgLoginUser'),
             'isUserLoggedIn' => $this->currentUser->isLoggedIn(),
             'isUserHasAdminRights' => $isUserHasAdminRights || $this->currentUser->isSuperAdmin(),
-            'baseHref' => $faqSystem->getSystemUri($this->configuration),
+            'baseHref' => $this->faqSystem->getSystemUri($this->configuration),
             'customCss' => $this->configuration->getCustomCss(),
             'version' => $this->configuration->getVersion(),
             'header' => str_replace('"', '', $this->configuration->getTitle()),
             'metaDescription' => $metaDescription ?? $this->configuration->get('seo.description'),
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$metaDescription'. (undefined)

(UnusedLocalVariable)